apps/desktop-wallet/turbo.json

```diff
@@ -0,0 +1,25 @@
+{
+  "extends": ["//"],
+  "pipeline": {
+    "start:electron": {
+      "dependsOn": ["^compile"],
+      "cache": false,
+      "persistent": true
+    },
+    "build:electron": {
+      "dependsOn": ["build"]
+    },
+    "build:electron:windows": {
+      "dependsOn": ["build"]
+    },
+    "build:electron:macos": {
+      "dependsOn": ["build"]
+    },
+    "build:electron:ubuntu": {
+      "dependsOn": ["build"]
+    },
+    "build:electron:linux:arm64": {
+      "dependsOn": ["build"]
+    }
```

Comment:
Here we say that each `build:electron*` task requires the `build` task to run first. The `build` task is defined in the root `turbo.json`.